### PR TITLE
Create malice.yml

### PR DESCRIPTION
**Pull request by Dodithegreat**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/chars/malice.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Terms
+Privacy
+Security
+Status
+Help
+Contact GitHub
+API
+Training
+Shop
+Blog
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/malice.yml]: Please remove all this random stuff at the bottom, you copied and pasted it by accident i think

Comment (Dodithegreat) [reply CommandHelper/LocalPackages/commands/chars/malice.yml]: kk

**Comment by Dodithegreat** [comment]
https://www.youtube.com/watch?v=PC6PDLTPWFk and her powers is extreme strength flight mindcontrol shapeshifting and can go to the underworld

**Comment by Dodithegreat** [comment]
4:56

**Pull request closed without merging**